packages/astro/CHANGELOG.md

```diff
@@ -140,65 +140,44 @@
 
 - [#5684](https://github.com/withastro/astro/pull/5684) [`a9c292026`](https://github.com/withastro/astro/commit/a9c2920264e36cc5dc05f4adc1912187979edb0d) Thanks [@bholmesdev](https://github.com/bholmesdev)! - Refine Markdown and MDX configuration options for ease-of-use.
```

Comment:
@delucis - I would replace lines 141 - 180 here with the same text I've used in MDX & markdown/remark (lines 43-85 in markdown/remark)

Those combine #5684 and #5769 because, in the end, `extendDefaultPlugins` was replaced by not just `gfm` but also a `smartypants` Boolean so we couldn't just ignore the SmartyPants saga entirely. Without 5769, we don't get any mention of the new `smartypants` config option.

Reply:
Ah great! Thought you did that last night, sorry, so didn't check.

Reply:
Done in [`4138675`](https://github.com/withastro/astro/pull/5951/commits/4138675a9fe1a80cc7ee1ccc2f33bebb550107a5)